src/test/java/core/basesyntax/service/impl/CsvWriterServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvWriterServiceTest {
+    private CsvWriterService csvWriterService;
+    private CsvReaderService csvReaderService;
+
+    @BeforeEach
+    void setUp() {
+        csvWriterService = new CsvWriterService();
+        csvReaderService = new CsvReaderService();
+    }
+
+    @Test
+    void write_file_ok() {
+        String content = "blabla";
+        String outputPath = "src/main/resources/output.csv";
```

Comment:
Next time do not make changes to main package from tests
```suggestion
        String outputPath = "src/test/resources/output.csv";
```